js/ui/grid_core/ui.grid_core.columns_view.js

```diff
@@ -825,6 +828,10 @@ export const ColumnsView = modules.View.inherit(columnStateMixin).inherit({
     _wrapTableInScrollContainer: function($table) {
         const $scrollContainer = $('<div>');
 
+        if(this.option('scrolling.useNative') === false || (this.option('scrolling.useNative') === 'auto' && !nativeScrolling)) {
```

Comment:
extract duplicated `this.option('scrolling.useNative')` to const

Reply:
Fixed in #28fb88b